### ISSUE TITLE
Can't find the USB stick device in UEFI mode

### ISSUE THREAD

**Issue by amirkarimi** [deadhead420/arch-linux-anywhere#125]
I just downloaded 2.2.4 iso and write to a USB disk using dd and booted in UEFI mode but got the following message:

```
Mounting '/dev/device/by-label/ARCH_201701' ... Waiting for 30 seconds ...
```

It can't find the USB device because its label is `AA224`:

```
ls /dev/device/by-label

AA224        (I don't remember what was here)
```

On the other hand I can't rename the partition in GParted. GParted doesn't load that partition! Why it looks for incorrect label as the ISO label is AA224?

**Comment by amirkarimi** [comment]
I see that it's fixed in a [commit](https://github.com/deadhead420/arch-linux-anywhere/commit/734afdc598696659faacce95ba97ca79c69f5463) but it seems that it the ISO Download link is still pointing to the incorrect version.

**Comment by FredBezies** [comment]
Try to build a new iso from git repository. Maybe it will work better?

**Comment by deadhead420** [comment]
Hey I have fixed this but the ISO needs to be rebuilt. Doing a few more tests and will release within a few hours.

**Comment by deadhead420** [comment]
I've just released version 2.2.5 of the ISO. This issue should no longer be present. Closing for now, thanks for the report.

**Comment by amirkarimi** [comment]
Thanks
